### PR TITLE
Update to 1.19

### PR DESCRIPTION
**Pull request by AlexandruGhergut**
I made this update for someone with wrist issues (can't hold W for too long). I tested it and it works but this is my first minecraft related contribution so let me know if I missed something.

P.S: Might need to create a new base branch 1.19 and merge this in there.

### CODE REVIEW

File: src/main/java/com/emonadeo/autorun/AutoRunModMenu.java [hunk 1]
```diff
@@ -18,22 +18,22 @@ public ConfigScreenFactory<Screen> getModConfigScreenFactory() {
     public Screen create(Screen screen) {
         ConfigBuilder builder = ConfigBuilder.create()
                 .setParentScreen(screen)
-                .setTitle(new TranslatableText("title." + AutoRun.MODID + ".config"));
+                .setTitle(Text.translatable(("title." + AutoRun.MODID + ".config")));
```
Comment (AlexandruGhergut) [inline src/main/java/com/emonadeo/autorun/AutoRunModMenu.java:21]: This is required due to the text changed described in the new version of Fabric [here](https://fabricmc.net/2022/06/07/119.html)

File: src/main/java/com/emonadeo/autorun/MovementDirection.java [hunk 2]
```diff
@@ -16,28 +16,28 @@ public enum MovementDirection {
     public KeyBinding getKeyBinding(MinecraftClient client) {
         switch (this) {
             default:
-                return client.options.keyForward;
```
Comment (AlexandruGhergut) [inline src/main/java/com/emonadeo/autorun/MovementDirection.java:19]: These changes are required due to the option changes described [here](https://fabricmc.net/2022/06/07/119.html)

**Comment by emonadeo** [comment]
Wait, I went to update it today and I swear I didn't see the PR. I checked yesterday but I didn't today. We both did _exactly_ the same, now I feel kinda stupid.

Welp. `¯\_(ツ)_/¯`

**Comment by AlexandruGhergut** [comment]
No problem 😆

**Pull request closed without merging**